controllers/main.go

```diff
@@ -1646,7 +1646,7 @@ func (controller *MainController) LoginPost(c web.C, r *http.Request) (string, i
 
 	log.Infof("Login POST from %v, email %v", remoteIP, user.Email)
 
-	if user.EmailVerified == 0 {
+	if (user.EmailVerified == 0 && controller.getNetworkName() != "testnet") {
```

Comment:
The brackets in this if condition are unnecessary

Reply:
The brackets have been removed